docs/layers/arc-layer.md

```diff
@@ -1,9 +1,42 @@
-## Arc Layer
+# Arc Layer
 
 The Arc Layer takes in paired latitude and longitude coordinated points and
 render them as arcs that links the starting and ending points.
 
-**Layer-specific Parameters**
+Inherits from all [Base Layer properties](/docs/layer.md).
 
-* `data` (array, required) array of objects: [{ position: {x0, y0, x1, y1},
-color }, ...]
+## Layer-specific Properties
+
+##### `strokeWidth` (Number, optional)
+
```

Comment:
There was no empty line in layer.md. Let's be consistent, whatever way looks best.